src/hotspot/share/utilities/hashtable.hpp

```diff
@@ -319,6 +319,27 @@ class KVHashtable : public BasicHashtable<F> {
     return NULL;
   }
 
+  // Look up the key.
+  // If an entry for the key exists, leave map unchanged and return a pointer to its value.
+  // If no entry for the key exists, create a new entry from key and value and return a
+  //  pointer to the value.
+  // *p_created is true if entry was created, false if entry pre-existed.
+  V* add_if_absent(K const& key, V const& value, bool* p_created) {
```

Comment:
Does it really need `const &` here? It looks inconsistent with `add(K,V)` and `lookup(K)` in the same class.

Reply:
I'll remove the `const&`. I copied that from resourceHash.hpp but it doesn't fit in with the rest of hashtable.hpp.